build/update-csv.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+YQ_MAJ_VERSION=${1:-4}
+installYQ() {
+    printf "Installing yq ..."
+    GO111MODULE=on go get github.com/mikefarah/yq/v4
```

Comment:
Can we bind to the specific enough for us `yq` version to prevent updates on some codegen steps

Reply:
unfortunately not, only major rel can be specified

Reply:
Not a good news, than we need go away from project folder temporary, i.e. /tmp install yq and than return back